app/components/events/view/overview/speaker-session.js

```diff
@@ -0,0 +1,23 @@
+import classic from 'ember-classic-decorator';
+import { classNames } from '@ember-decorators/component';
+import { computed } from '@ember/object';
+import Component from '@ember/component';
+
+@classic
+@classNames('ui', 'fluid', 'card')
+export default class SpeakerSession extends Component {
+  @computed('data.orderStat.tickets')
+  get tickets() {
```

Comment:
what is the use of all these computed prop. Where are they used?

Reply:
Sorry i did this by mistake
and i ll do above changes.thanks for the review

Reply:
no need of this file.
Delete it

Reply:
And also make sure that it is working good in mobile view